Kickstarter-iOS/Views/Controllers/EmailVerificationViewController.swift

```diff
@@ -0,0 +1,182 @@
+
+  private lazy var messageLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var contentVStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var contentHStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var imageView: UIImageView = { UIImageView(frame: .zero) }()
+  private lazy var footerLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var footerStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var resendButton: UIButton = { UIButton(type: .custom) }()
+  private lazy var rootStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var skipButton: UIButton = { UIButton(type: .custom) }()
+  private lazy var titleLabel: UILabel = { UILabel(frame: .zero) }()
+
+  private let viewModel: EmailVerificationViewModelType = EmailVerificationViewModel()
+
+  // MARK: - Lifecycle
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    self.configureSubviews()
+  }
+
+  // MARK: - Styles
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self.view
+      |> \.backgroundColor .~ .white
+
+    _ = self.rootStackView
+      |> rootStackViewStyle
+
+    _ = self.contentHStackView
+      |> contentHStackViewStyle
+
+    _ = self.contentVStackView
+      |> contentVStackViewStyle
+
+    _ = self.imageView
+      |> UIImageView.lens.image .~ Library.image(named: "email-icon-light")
+
+    _ = self.titleLabel
+      |> titleLabelStyle
+
+    _ = self.messageLabel
+      |> messageLabelStyle
+
+    _ = self.skipButton
+      |> skipButtonStyle
+
+    _ = self.footerLabel
+      |> footerLabelStyle
+
+    _ = self.resendButton
+      |> resendButtonStyle
+  }
+
+  private func configureSubviews() {
+    _ = (self.rootStackView, self.view)
+      |> ksr_addSubviewToParent()
+      |> ksr_constrainViewToMarginsInParent()
+
+    _ = ([self.contentHStackView, self.footerStackView], self.rootStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    _ = ([self.contentVStackView], self.contentHStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    _ = ([self.imageView, self.titleLabel, self.messageLabel, self.skipButton], self.contentVStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    _ = ([self.footerLabel, self.resendButton], self.footerStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    self.contentVStackView.setCustomSpacing(Styles.grid(5), after: self.imageView)
+  }
+
+  // MARK: - View model
+
+  internal override func bindViewModel() {
+    super.bindViewModel()
+  }
+}
+
+// MARK: - Styles
+
+private let skipButtonStyle: ButtonStyle = { (button: UIButton) in
+  button
+    |> UIButton.lens.titleLabel.textAlignment .~ .center
+    |> UIButton.lens.titleLabel.font .~ UIFont.ksr_subhead().bolded
+    |> UIButton.lens.titleColor(for: .normal) .~ .ksr_green_500
+    |> UIButton.lens.title(for: .normal) %~ { _ in
+      localizedString(
+        key: "Ill_do_this_later",
```

Comment:
Should we be pulling this and the other new strings from the server? I'm not sure what the process is to set that up.

Reply:
Yeah I added them here for you to review: https://github.com/kickstarter/kickstarter/pull/20912

Once that merges they'll be translated and then we will re-generate our strings with `make strings`.

I've just put them in manually for now so that the UI renders correctly. I'm using the `localizedString` function so that localization still works correctly once they come in and these strings are easy to find to update them to their `Strings.*` counterpart.